code/game/objects/items/food/_food.dm

```diff
@@ -71,3 +72,16 @@
 		AddElement(/datum/element/food_trash, trash_type)
 	return
 
+///This proc handles grillable components, overwrite if you want different grill results etc.
+/obj/item/reagent_containers/food/proc/make_grillable()
```

Comment:
you likely want to add the `make_grillable()` proc on *both* `/obj/item/food/` and `/obj/item/reagent_containers/food/` paths. This one here is with other `/obj/item/food` code; check the file `code/modules/food_and_drinks/food.dm` where you can see `make_bakeable()` is defined there as well.

```suggestion
/obj/item/food/proc/make_grillable()
```